netann/chan_status_manager.go

```diff
@@ -21,9 +21,9 @@ var (
 	// ErrInvalidTimeoutConstraints signals that the ChanStatusManager could
 	// not be initialized because the timeouts and sample intervals were
 	// malformed.
-	ErrInvalidTimeoutConstraints = errors.New("active_timeout + " +
-		"sample_interval must be less than or equal to " +
-		"inactive_timeout and be positive integers")
+	ErrInvalidTimeoutConstraints = errors.New("chan-enable-timeout + " +
+		"chan-status-sample-interval <= chan-disable-timeout " +
```

Comment:
```suggestion
		"chan-status-sample-interval must <= chan-disable-timeout " +
```

Maybe add a `must` to point what's needed otherwise it can be confusing?

Reply:
Added for more clarity